pkg/deploy/rp.go

```diff
@@ -484,37 +498,65 @@ func (g *generator) vault() *arm.Resource {
 			},
 			AccessPolicies: &[]mgmtkeyvault.AccessPolicyEntry{},
 		},
-		Name:     to.StringPtr("[parameters('keyvaultName')]"),
+		Name:     to.StringPtr("[concat(parameters('keyvaultPrefix'), '-clusters')]"),
 		Type:     to.StringPtr("Microsoft.KeyVault/vaults"),
 		Location: to.StringPtr("[resourceGroup().location]"),
+		Tags: map[string]*string{
+			"vault": to.StringPtr("clusters"),
```

Comment:
I'm worried about the growing number of magic values (especially very generic ones) spread across the codebase. I'm afraid that in couple of months time we will not be able to distinguish `"clusters"` values from each other. Is it just a tag or is it important in logic?

Maybe we should start moving these into constants to be easier find relevant places?

Reply:
Agreed.  Pushed an additional commit here as a starting point.